src/Adapters/Logger/WinstonLoggerAdapter.js

```diff
@@ -0,0 +1,106 @@
+import { LoggerAdapter } from './LoggerAdapter';
+import { logger, addTransport } from '../../logger';
+
+let currentDate = new Date();
+
+let simpleCache = {
+  timestamp: null,
+  from: null,
+  until: null,
+  order: null,
+  data: [],
+  level: 'info',
+};
+
+// returns Date object rounded to nearest day
+let _getNearestDay = (date) => {
+  return new Date(date.getFullYear(), date.getMonth(), date.getDate());
+}
+
+// returns Date object of previous day
+let _getPrevDay = (date) => {
+  return new Date(date - MILLISECONDS_IN_A_DAY);
+}
+
+// returns the iso formatted file name
+let _getFileName = () => {
+  return _getNearestDay(currentDate).toISOString()
+}
+
+// check for valid cache when both from and util match.
+// cache valid for up to 1 minute
+let _hasValidCache = (from, until, level) => {
+  if (String(from) === String(simpleCache.from) &&
+    String(until) === String(simpleCache.until) &&
+    new Date() - simpleCache.timestamp < CACHE_TIME &&
+    level === simpleCache.level) {
+    return true;
+  }
+  return false;
+}
+
+// check that log entry has valid time stamp based on query
+let _isValidLogEntry = (from, until, entry) => {
+  var _entry = JSON.parse(entry),
+    timestamp = new Date(_entry.timestamp);
+  return timestamp >= from && timestamp <= until
+    ? true
+    : false
+};
+
+export class WinstonLoggerAdapter extends LoggerAdapter {
+  info() {
+    return logger.info.apply(undefined, arguments);
+  }
+
+  error() {
+    return logger.error.apply(undefined, arguments);
+  }
+
+  addTransport(transport) {
```

Comment:
i really just wanted to expose addTransport on logger, but my require/import foo isn't up to snuff.  Glad to take another crack at it now that I have a working prototype.